definitions/npm/react-apollo_v2.x.x/flow_v0.58.x-/react-apollo_v2.x.x.js

```diff
@@ -234,13 +234,28 @@ declare module 'react-apollo' {
     pollInterval?: number,
     notifyOnNetworkStatusChange?: boolean,
     fetchPolicy?: FetchPolicy,
-    errorPolicty?: ErrorPolicy,
+    errorPolicy?: ErrorPolicy,
     ssr?: boolean,
     displayName?: string,
     delay?: boolean,
     context?: {[string]: any}
   }> {}
 
+  declare type SubscriptionResult<TData> = {
+    loading: boolean,
+    data?: TData,
+    error?: ApolloError,
+  }
+
+  declare type SubscriptionProps<TData> = {
+    subscription: DocumentNode,
+    variables?: { [string]: any },
+    shouldResubscribe?: boolean | (SubscriptionProps<TData>, SubscriptionProps<TData>) => boolean,
+    children: (result: SubscriptionResult<TData>) => React$Node,
+  }
+
+  declare export class Subscription<TData> extends React$Component<SubscriptionProps<TData>> {}
```

Comment:
I'm pretty sure you don't need to export it explicitly (in the same way you don't have to `declare export type`), can you verify?

Reply:
This is consistent with the way `Query` and `Mutation` are exported. And if I remove `export` from them I get the error like:

```
Cannot import Query because there is no Query export in react-apollo.
```

Reply:
Got it.